src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -474,53 +474,42 @@ def process_scores_(
         self.num_entities = num_entities
 
 
+TARGET_TO_KEYS = {target: [TARGET_TO_INDEX[c] for c in COLUMN_LABELS if c != target] for target in COLUMN_LABELS}
```

Comment:
maybe also move this to the constants.py and make sure we clean up unused imports

Reply:
https://github.com/pykeen/pykeen/pull/1079/commits/0f03380fc68c19b960010078a601b05f264e0cb3